extensions/shared/wrap-paid-block.jsx

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
You'll need to add this and other new files to SVN patch so that this gets synced to .com in D29733-code. Now the build there is failing due missing files.

Reply:
Yep, see PR desc:

> Once this PR has approval, I'll manually sync the newly created files over to the WP.com counterpart, D29733-code. Doing that now doesn't make sense since they'll be removed by Fusion each time this PR changes.

Reply:
And yeah, IIRC, I've asked Crew before about having Fusion auto-sync newly added files. The answer was: It's complicated, can't do it yet, sry (again, IIRC).

Reply:
Ahh, sorry, missed that 👍